app/views/content/shared/block-promos/_adviser_findske.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="row">
+  <section class="container">
+    <section class="image-blocks">
+      <%= render CallsToAction::AdviserBlockComponent.new %>
```

Comment:
Do we want to update the adviser text to be `Maximise.....` to be consistent with the rest of the section?